caddyhttp/httpserver/plugin.go

```diff
@@ -25,6 +25,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/mholt/caddy/caddyhttp/staticfiles"
+
```

Comment:
Delete whitespace and gofmt to rearrange imports

Reply:
Done